apps/trace_clearMemory/trace_kernel_clearMemory.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-License-Identifier: GPL-2.0-only
```

Comment:
This one is missing a `Copyright` line. Would be fine to put in `Copyright 2021 Kry10` (or yourself)